python/lsst/ip/isr/assembleCcdTask.py

```diff
@@ -195,13 +197,15 @@ def assembleCcd(self, assembleInput):
         ccd = None
         if hasattr(assembleInput, "has_key"):
             # Get a detector object for this set of amps
-            ccd = assembleInput.itervalues().next().getDetector()
+            ccd = assembleInput.values().next().getDetector()
```

Comment:
Hmm. Is that a real `.next()` method in the iterator "now known as `next()`" way? Or is that a method that just happens to be called `next()`?